netcheck.sh

```diff
@@ -1,10 +1,6 @@
 #!/bin/bash
 
-################################################################################
-##               Netcheck - Simple internet connection logging                ##
-##               https://github.com/TristanBrotherton/netcheck                ##
-##                                       -- Tristan Brotherton                ##
-################################################################################
+
```

Comment:
This was amazing

Reply:
Another comment

Reply:
I ate breakfast

Reply:
I really like this

Reply:
Can you add a some code here to define a new function


```suggestion
#!/bin/bash

check_connection() {
    # Function body will be implemented later
    echo "Checking connection..."
}
```